Add athena example (#260)
flyteorg/flytesnacks: cookbook/integrations/aws/athena/README.rst

######
Athena
######

Executing Athena Queries
=======================

> Flyte backend can be connected with athena. Once enabled it allows you to query AWS Athena service (Presto + ANSI SQL Support) and retrieve typed schema (optionally).

Reviewer: sorry
```suggestion
Flyte backend can be connected with athena. Once enabled, it allows you to query AWS Athena service (Presto + ANSI SQL Support) and retrieve typed schema (optionally).
```

Author: done

######
Athena
######

Executing Athena Queries
=======================

> Flyte backend can be connected with athena. Once enabled it can allow you to query a athena service (e.g. Qubole) and retrieve typed schema (optionally).

Reviewer: ```suggestion
Flyte backend can be connected with athena. Once enabled it allows you to query AWS Athena service (Presto + ANSI SQL Support) and retrieve typed schema (optionally).
```

> This section will provide how to use the athena Query Plugin using flytekit python

Reviewer: ```suggestion
This section provides a guide on how to use the AWS Athena Plugin using flytekit python.
```

Author: done

=======

> :nosearch:

Reviewer: What does this do?

Author: removed

Installation
------------

To use the flytekit athena plugin simply run the following:

.. prompt:: bash

    pip install flytekitplugins-athena

> No Need of a dockerfile

Reviewer: `of` or `for`?

No idea....

------------------------

> This plugin is purely a spec and since SQL is completely portable has no need to build a container. Thus this plugin examples do not have any Dockerfile

Reviewer: ```suggestion
This plugin is purely a spec and since SQL is completely portable, it has no need to build a container. Thus this plugin examples do not have any Dockerfiles.
```

Configuring the backend to get athena working

> -------------------------------------------

Reviewer: ```suggestion
----------------------------------------------
```

.. todo:

> This is coming soon. 🛠

Reviewer: How soon? :P

Athena
======

Executing Athena Queries
------------------------

Flyte backend can be connected with Athena. Once enabled it can allow you to query a athena service (e.g. Qubole) and retrieve typed schema (optionally).
This section will provide how to use the athena Query Plugin using flytekit python

Installation
------------

To use the ``flytekit-athena`` plugin simply run the following:

.. prompt:: bash

    pip install flytekitplugins-athena

.. NOTE::

    This plugin is purely a spec since SQL is completely portable has no need to build a container, therefore the plugin
    examples do not have an associated Dockerfile.

Configuring the backend to get athena working
---------------------------------------------

.. NOTE::

>     Coming soon. 🛠

Reviewer 2: are we vetoing any user-facing "coming soon" language?

an alternative I'd suggest is an inline comment ``.. TODO: add "Configuring the backend to get athena working`` that doesn't show up on the docs site, assuming that this section will be added as part of another PR

Author: updated, thanks!
